castai/resource_node_template.go

```diff
@@ -605,7 +607,9 @@ func getNodeTemplateByName(ctx context.Context, data *schema.ResourceData, meta
 	nodeTemplateName := data.Id()
 
 	log.Printf("[INFO] Getting current node templates")
-	resp, err := client.NodeTemplatesAPIListNodeTemplatesWithResponse(ctx, clusterID)
+	resp, err := client.NodeTemplatesAPIListNodeTemplatesWithResponse(ctx, clusterID, &sdk.NodeTemplatesAPIListNodeTemplatesParams{
+		IncludeDefault: lo.ToPtr(false),
```

Comment:
This PR seems to have unrelated Node Template changes

Reply:
API changes forced this one, can't do anything about it - could maybe separate out as another PR, but seems like a small change to pack together.